includes/Elements/Section/Section.php

```diff
@@ -515,6 +533,33 @@ public function before_render( $options ) {
 	 * @return void
 	 */
 	public function render( $options ) {
+		$masks = $options->get_value( 'shapes' );
```

Comment:
Can the code related to masks be moved as a static class method to Mask.php? By doing this, we can easily add masks to other elements without copying the same code again.